lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -715,4 +717,50 @@ main .attribute-access-type {
   font-family: var(--font-code);
 }
 
+@media (max-width: 1023px) {
+  body {
+    font-size: 14px;
```

Comment:
Any particular reason not to use a bigger font size like the browser default 16px? I know it takes more space, but reading on mobile is harder (in particular for us, wearing glasses 😅)